applications/rag/frontend/variables.tf

```diff
@@ -66,4 +66,69 @@ variable "google_service_account" {
   type        = string
   description = "Google Service Account name"
   default = "frontend-gcp-sa"
+}
+
+variable "k8s_ingress_name" {
```

Comment:
will the frontend support a no auth version like jupyterhub? i guess not, since we don't have any dummy/placeholder login logic